plugins/mcreator-core/blockly/js/field_ai_condition_selector.js

```diff
@@ -49,8 +49,7 @@ class FieldAiConditionSelector extends Blockly.Field {
                 javabridge.openAIConditionEditor(this.getValue() || 'null,null', { // If somehow the value is missing, pass 'null,null'
                     'callback': function (data) {
                         const group = Blockly.Events.getGroup();
-                        if (!group)

```

Comment:
>  Before it looked cleaner

But now it's different from how events are handled in core Blockly.

Reply:
It does the same thing to my knowledge and looks cleaner. It is also how you implemented this initially